File: JSBASIC/99_browser_name.js
// 99	99_browser_name.js	Display the browser name based on a code input using switch.

let code = "CH";
console.log("Browser Code:", code);
switch (code.toUpperCase()) {
    case "CH": console.log("Browser: Google Chrome"); break;
    case "FF": console.log("Browser: Mozilla Firefox"); break;
    case "IE": console.log("Browser: Internet Explorer"); break;
    case "SF": console.log("Browser: Safari"); break;
    case "ED": console.log("Browser: Microsoft Edge"); break;
    case "OP": console.log("Browser: Opera"); break;
    default:   console.log("Unknown Browser Code.");
}

File: JavaBasic/99_browser_name.java
// 99	99_browser_name.java	Display the browser name based on a code input using switch.
public class _99_browser_name {
    public static void main(String[] args) {
        String code = "CH";
        switch (code.toUpperCase()) {
            case "CH": System.out.println("Google Chrome"); break;
            case "FF": System.out.println("Mozilla Firefox"); break;
            case "IE": System.out.println("Internet Explorer"); break;
            case "SF": System.out.println("Safari"); break;
            case "ED": System.out.println("Microsoft Edge"); break;
            default:   System.out.println("Unknown Browser.");
        }
    }
}
